README.md

```diff
@@ -119,21 +119,22 @@ The `filtering/check_glpi_reservation.py` script queries the GLPI deployment for
 The `filtering/check_glpi_reservation.py` script queries the GLPI deployment for computers and prints a digest of computer fields. For usage information see the help message provided by the script.
 
 ### create_reservation_wrapper.py
-This is the recommended workflow for creating reservations using the REST API. One has the ability to manually manage reservations via the GUI (fine for an individual machine or to view the calendar), but for convenience there is an included wrapper around `reservation/create_glpi_reservation.py` which can take in a YAML file defining reservations of machines for given Jira tags (or other work item IDs). The wrapper takes in the API token as well as the reservation YAML file. Required for proper function are the Jira Epic tag, username for which to reserve the machine, start and end time in "YYYY-MM-DD HH:MM:SS" format, and servers by hostname. The global comment field will be added to the comment field below the Jira Epic tag if specified (omitted if set to None). If a server contains None (~) then it will use the global Jira Epic tag fields, otherwise they will be overwritten for that specific machine. Below is an explanation of the required YAML structure (also see `reservation/reservation_example.yaml`):
+This is the recommended workflow for creating reservations using the REST API. One has the ability to manually manage reservations via the GUI (fine for an individual machine or to view the calendar), but for convenience there is an included wrapper around `reservation/create_glpi_reservation.py` which can take in a YAML file defining reservations of machines. The wrapper takes in the API token as well as the reservation YAML file. Required for proper function are the username for which to reserve the machine, start and end time in "YYYY-MM-DD HH:MM:SS" format, and servers by hostname. You can optionally include a Jira Epic, which will be included as a comment if specified. The global comment field will be added to the comment field below the Jira Epic tag if specified (omitted if set to None). If a server contains None (~) then it will use the global fields, otherwise they will be overwritten for that specific machine. Below is an explanation of the required YAML structure (also see `reservation/reservation_example.yaml`):
 <pre>
-JIRA-0000:
-  username:example_user
-  start:"2021-09-30 23:59:59"
-  end:"2021-10-30 23:59:59"
-  comment:~
-  servers:
-    hostname-1:
-      ~
-    hostname-2:
-      username:overwritten_username
-      start:"2021-10-01 23:59:59"
-      end:"2021-10-31 23:59:59"
-      comment:"a comment overwriting the above global comment"
+username:example_user
+start:"2021-09-30 23:59:59"
+end:"2021-10-30 23:59:59"
+epic: "JIRA-0000" (optional)
```

Comment:
Should the other fields that are optional also be marked as such? (comment is I believe optional, epic under a specific machine, etc.)

Reply:
agreed, just made a change